Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+ENV LC_ALL=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /app
+COPY . /app
+
+RUN \
```

Comment:
@mehdisadeghi as a side note, I guess it would probably be better for everyone to just have a pyosmium docker image. Instead of replicating this work, everyone could use it as base and simply run their python scripts on top. I guess it would be trivial to run a python script once you have those things installed.

What do you think?

Reply:
@carocad I think having a pyosmium image has little benefits. pyosmium is already packaged in Ubuntu repositories so it is relatively easy to install. The reason I added this Dockerfile was to package osmtogtfs and pyosmium as an executable docker image. To be honest I'm not sure whether this is a useful thing at all.

Reply:
@mehdisadeghi I have my doubts as well but I thought that you have already checked it and concluded that it was needed.

I just went with this approach because I recently tried to install pyosmium in a Mac laptop and it didnt work. However I might be doing something wrong since I only tried 5 minutes jejeje

Reply:
> I recently tried to install pyosmium in a Mac laptop and it didnt work. 

Actually this was the main purpose. To help people who have trouble installing pyosmium but have access to docker. This way people can directly interact `osmtogtfs` without having to deal with pyosmium's installation (which might need dev tools installed).